### DISCUSSION TITLE
2023-06-30

### DISCUSSION THREAD

**Discussion by aak2075** [woowacourse-teams/2023-haru-study#2] [daily_meeting]
### 지난 KPT 종합

K - 회의 시간 정하기라는 action을 실천함으로 문제를 해결하려고 했다
P - 딱딱한 회의, 팀원들간 아이디어에 대한 방향성이 다름
T - 생각 동기화, 칭찬하기

### Action Item

생각 동기화 - 한 번의 회의 사이클(50분)을 시작하기 전 목표를 설정하고 회의를 시작한다.
칭찬하기 - 면전에 따봉을 날리자

### 오늘 할 일

- 그라운드 룰 정하기
- Github milestone 설정
- 문서 작성 (이슈, 디스커션)템플릿 및 작성 가이드 문서화
- 이슈 템플릿, pr 템플릿, 디스커션 템플릿, 프로젝트, 칸반보드
- 프론트 백엔드에 대한 코드 컨벤션 문서화

### 내일 할 일

- 와이어 프레임 스케치 - 7.3
- 구체화된 구현 기능 목록 명세 - 7.3

**Comment by aak2075** [comment]
### 체크인

10점 - 불금 좋아요

### 체크아웃

10점 - 팀 빌딩 게임 미션이 재미있었다.

### KPT

K : FigJam의 포스트잇으로 다양한 의견 교환
P : 그다지 중요하지 않은 안건을 의사 결정에 너무 많은 시간을 쓴다는 느낌이 있음.
T : 중요도에 따라서 적당히 의사결정을 하면 좋을 것 같다.

**Comment by yeopto** [comment]
### 체크인

6점 - 원래 너무 피곤해서 3점이었는데 마코와 개발이야기해서 +3

### 체크아웃

8점 - 졸림이 사라지고 한 주도 뿌듯하게 보낸 것 같아서 +2

### KPT

K: 회고를 한 후 좋은 의견에 대해선 얼굴에 따봉날리기라는 action item을 정해서 행동한 것이 좋았던 것 같다.
P: 브레인 스토밍 방식으로 의견 취합을 하고있는데, 모은 의견을 팀원들이 다 동의하는 것에 대해서만 간추리려하다보니 회의가 너무 길어진다.
T: 의견 취합 방법에 대해서 새로운 방안을 찾아보면 어떨까?

**Comment by MoonJeWoong** [comment]
### 체크인
7점- 아침에 1시간 늦잠을 자버려서 3점 깎았읍니당... 오늘 하루 열심히 해서 10점 체크아웃 만들어 봅시당 

### 체크아웃
8점 - 그라운드 룰 완성해서 기분은 좋아졌지만 의사결정 효율성에 아쉬운 점이 남는 것 같습니다.

### KPT
K - 칭찬할 부분을 확실하게 칭찬하자는 액션 아이템을 잘 지킨게 좋았습니다.
P - 의사결정 과정에서 결과물에 비해 드는 여러가지 비용이 너무 컸습니다.
T -  의사결정 과정을 시각적으로 구조화하면 어떨지 제안합니다.

**Comment by jaehee329** [comment]
### 체크인

5점 - 어제 한시간 늦게 잤더니 하품 엄청 나오네요. 주말에 몰아서 자야지...

### 체크아웃

7점 - 보람차네요 그래도. 주말이다~

### KPT
K: 다 같이 회의의 방향성을 잃지 않으려고 회의 중간중간 엇나가는 의견들을 잘 바로잡는 것 같다. 계속 하자!
P: 중요도가 높지 않은 부분들에 대해서도 세심하게 결정하다보니 속도가 느리다
T: 그 날 할 일의 중요도를 아침에 살펴보고 파급력이 낮은 작업은 빠르게 결정할 수 있는 수단을 쓰자. (ex) 한 명이 주도적으로 의견 제안하고 넘어가기 등)

P2: KPT의 T가 구체적이지 않아 조금 아쉽다.
T2: T에서는 구체적인 해결책을 제안할 수 있는 방식으로 서술하자.

**Comment by nlom0218** [comment]
### 체크인

8점 - 어제도 도시락통을 두고감. 겁나 찜찜함...

### 체크아웃

9점 - 살짝 피곤하지만 오늘 필수적으로 해야 할 일을 잘 마무리하였다. 팀도 조금씩 맞춰나가는 기분이 든다.

### KPT

K: 시간을 제한을 두어 빠르게 회의를 진행하였다. 투표를 통해 의견 수렴이 잘 이루어졌다.
P: 개인적으로 말이 계속 빨라지는 것 같다. 급한 마음이 들지만 천천히 하도록 하자.
T: 오늘 다수결로 투표를 많이 진행했는데, 이 방법이 앞으로 쭉 이어나가도 되는 것인지 의논이 필요해 보인다.

**Reply by jaehee329** [reply]
다수결이라는 방식이 위험하죠~

**Comment by woosung1223** [comment]
### 체크인
8점 - 오늘 금요일이고 본가 가는 날이라 기분이 좋습니다잉

### 체크아웃
10점 - 엽토의 댄스가 획기적이라 10점 드립니다

### KPT
K: 회의 흐름을 투표를 통해 모두가 주도하는 방식이라 좋았다
P: 계속 투표를 하는 방식이 조금은 회의가 루즈해지는 느낌이 들었다
T: 기술 관련 결정이 아닌, 간단한 논의사항이라면 미리 의견을 준비해와서 회의에서는 취합만 하면 어떨까?

**Reply by nlom0218** [reply]
테오의 춤 더 보고싶은데?

**Reply by nlom0218** [reply]
미리 의견을 준비한다면 브레인스토밍 과정이 훨씬 짧아 질 것 같네 :)

**Comment by woo-jk** [comment]
### 체크인

9점 - 어제 약 30분정도 일찍 자서 덜피곤하다. 그리고 오늘은 금요일 ㅎㅎ

### 체크아웃
8점 - 금요일이라 10, 회의가 너무 지쳐 -3,  그래도 보람차서 +1

### KPT
K: 피그잼을 통해 회의를 진행해봤는데, 시각적인 결과가 확실하게 남아서 좋다.
P: 항상 회의가 짧게 끝나지 않는다.
T: 여러 가지 회의 방식을 시도해보면서 회의 시간도 줄이고, 좋은 결과도 얻을 수 있는 회의 방식을 찾아보자
